client/src/components/Login.jsx

```
@@ -66,6 +66,7 @@ const Login = () => {
       return;
     }
 
+    const captchaToken = recaptchaRef.current.getValue();
     const captchaToken = recaptchaRef.current.getValue();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'captchaToken'. Consider to delete it or rename it.

'captchaToken' is defined here:


(lint/suspicious/noRedeclare)